Intersect.Client.Core/Interface/Game/GameInterface.cs

```diff
@@ -92,21 +92,22 @@ public partial class GameInterface : MutableInterface
 
     public PlayerStatusWindow PlayerStatusWindow;
 
-    private SettingsWindow GetOrCreateSettingsWindow()
+    private SettingsWindow SettingsWindow
     {
-        _settingsWindow ??= new SettingsWindow(GameCanvas)
+        get
         {
-            IsVisible = false,
-        };
-
-        return _settingsWindow;
+            return _settingsWindow ??= new SettingsWindow(GameCanvas)
+            {
+                IsVisible = false,
+            };
+        }
     }
 
     public GameInterface(Canvas canvas) : base(canvas)
     {
         GameCanvas = canvas;
-        EscapeMenu = new EscapeMenu(GameCanvas, GetOrCreateSettingsWindow) {IsHidden = true};
-        SimplifiedEscapeMenu = new SimplifiedEscapeMenu(GameCanvas, GetOrCreateSettingsWindow) {IsHidden = true};
+        EscapeMenu = new EscapeMenu(GameCanvas, SettingsWindow) {IsHidden = true};
+        SimplifiedEscapeMenu = new SimplifiedEscapeMenu(GameCanvas, SettingsWindow) {IsHidden = true};
```

Comment:
tl;dr: Revert the changes in this file

`SettingsWindow` is a bit heavy and I intentionally wanted to defer creation until it's needed. To that end, passing a delegate was necessary and the getter, while on the surface is doing the same thing, isn't actually having the same effect.

In this case it's creating only one, but it's creating it immediately in the constructor of `GameInterface`. With the methods, it was only ever created when `_settingsWindowProvider()` was invoked.